src/domain/shared/components/ActivityLog/ActivityComponent.tsx

```diff
@@ -63,9 +63,10 @@ export const ActivityComponent: FC<ActivityLogComponentProps> = ({ activities =
   const display = useMemo(() => {
     return (
       <>
-        {activities.map(activity => (
-          <ActivityViewChooser activity={activity} journeyLocation={journeyLocation} />
-        ))}
+        {activities.map(activity => {
+          const key = new Date(activity.createdDate).getTime();
```

Comment:
the point of the key is to not change over time so React does not re-render the element

Reply:
surely the key should just be the activity.id?

Reply:
we sorted it out. The problem was that id wasn't fetched with the query.

Reply:
Super :)